digitalstrategie/assets/scss/components/_faq_block.scss

```diff
@@ -0,0 +1,13 @@
+.accordion {
+    background-color: $white;
```

Comment:
I guess, if you leave out the background colour out, it is also white? I think instead of wetting it white here, we should so that, so that we use the background colour we use elsewhere. (So that if we use a light gray or something that is not overwritten)

Reply:
yes, will do